easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -630,25 +630,33 @@ def _generate_template_values(self, ignore=None, skip_lower=True):
                 del self.template_values[k]
 
     @handle_deprecated_or_replaced_easyconfig_parameters
+    def __contains__(self, key):
+        """Check whether easyconfig parameter is defined"""
+        return key in self._config
+
+    @handle_deprecated_or_replaced_easyconfig_parameters
     def __getitem__(self, key):
-        """
-        will return the value without the help text
-        """
-        value = self._config[key][0]
+        """Return value of specified easyconfig parameter (without help text, etc.)"""
+        value = None
+        if key in self._config:
```

Comment:
we could just use `key in self`, but then it's rather disconnected from the bit that uses `self._config`...